Gaijinizer patch/CommonEvents_122.txt

```diff
@@ -119,145 +128,153 @@ something on barghest ...#MTLed
-all right? You are not human anymore♪
-Just me faithful to me rex.#MTLed
+Do you get it now? You're no longer human. ♪
+You're now just my faithful doggy.#Ron
 > END STRING
 
 > BEGIN STRING
 よかったじゃない♡
 バーゲストからそれをもらうって事は、相当好きなんでしょ？
 バーゲストが…
 > CONTEXT: CommonEvents_122/96/Dialogue/enemy2
-I'm not happy. You really like getting it from
-barghest, right? barghest is ...#MTLed
+Aren't you happy? ♡ You really like getting it from
+barghests, right? In fact...#Ron
 > END STRING
 
 > BEGIN STRING
 だったら\\N[1]クン…。
 
 アンタもイヌになっちゃえば？
 > CONTEXT: CommonEvents_122/100/Dialogue/enemy2
-If it were \\N[1] Kun .... Should you become a
-dog too?#MTLed
+Yes, maybe \\N[1]-kun should become a dog too.#Ron
 > END STRING
 
 > BEGIN STRING
 キャハハハハハハハハハ！！
 
 おもしろいこと考えるねぇ！！エンプちゃん！
 > CONTEXT: CommonEvents_122/106/Dialogue/enemy2
-Heh hahahahahahahahaha!!Think about something
-interesting!!Empuse-chan!#MTLed
+Kyaahahahahahahahahaha!!
+
+You have interesting ideas, Empuse-chan!#Ron
 > END STRING
 
 > BEGIN STRING
 よーし、その案、採用しちゃおっと♪
 > CONTEXT: CommonEvents_122/110/Dialogue/enemy2
-Good luck, that plan, adopted♪#MTLed
+Aaalrighty, we'll go with that. ♪#Ron
 > END STRING
 
 > BEGIN STRING
 ふふふーん♪
 
 \\N[1]クンの魂は魔力があるぶん弄りやすいなぁ♡
 > CONTEXT: CommonEvents_122/114/Dialogue/enemy2
-Fufufu♪. The soul of \\N[1] Kun is easy to beat
-with magic#MTLed
+Fufufu. ♪
+
+\\N[1]-kun's soul is pretty easy to mold with
+magic. ♪#Ron
 > END STRING
 
 > BEGIN STRING
 よーし！できたぁ♪
 
 こっちのほうが、\\N[1]クンにはお似合いの姿だよ♡
 > CONTEXT: CommonEvents_122/119/Dialogue/enemy2
-Good! It was possible♪. This one is a figure
-that looks good for \\N[1] Kun#MTLed
+Aaalright, all done! ♪
+
+This form really suits you, \\N[1]-kun. ♡#Ron
 > END STRING
 
 > BEGIN STRING
 キャハハハハハ！！
 
 アタシ、オスのバーゲストとか初めて創ったかもぉ♪
 > CONTEXT: CommonEvents_122/138/Dialogue/enemy2
-Heh hahahahaha!! 'I', male barghest or maybe the
-first time created♪#MTLed
+Kyahahahahaha!!
+
+This might be the first time I've made a male
+barghest. ♪#Ron
 > END STRING
 
 > BEGIN STRING
 ほらほらぁ…♡
 
 せっかくイヌになったんだから、泣いてみなよぉ？
 > CONTEXT: CommonEvents_122/142/Dialogue/enemy2
-Here you see...♡. You're a dog so why don't you
-cry?#MTLed
+Go on then...♡
+
+You're a doggy now, why don't you try barking?#Ron
 > END STRING
 
 > BEGIN STRING
 ワ…ワゥ……
 > CONTEXT: CommonEvents_122/147/Dialogue/kazuya-2
 > CONTEXT: CommonEvents_122/185/Dialogue/kazuya-2
-Wah Wah Wah#MTLed
+W- wo-...#Ron
 > END STRING
 
 > BEGIN STRING
 声が小さい
 > CONTEXT: CommonEvents_122/149/Dialogue/enemy3
-Voice is small#MTLed
+I can't hear you.#Ron
 > END STRING
 
 > BEGIN STRING
 ワウッ！！！ワウワウッ！！
 > CONTEXT: CommonEvents_122/152/Dialogue/kazuya-2
-Waoof!!!Wawooh!!#MTLed
+WOOF!!! WOOOOF!!#Ron
 > END STRING
 
 > BEGIN STRING
 あっれー？\\N[1]わんちゃん
 
 アタシに蹴られて興奮してるの？とんだマゾ犬だねぇ。
 > CONTEXT: CommonEvents_122/154/Dialogue/enemy3
-Oh?\\N[1] Wanchan. Are you kicked by 'I' and
-Aroused?You're a masochist dog.#MTLed
+Oh? Do you get aroused when I kick you my little
+\\N[1] doggy? What a masochistic dog you are.#Ron
```

Comment:
Minor things, I'd phrase it like this:
>_**Oh...? Does my little doggy get aroused when
I kick him? What a masochistic dog you are.**_
(Tried keeping MC-dog suffix, but it's honestly really stiff in English).